play-media-native/features/Screens/AthleteImages.tsx

```diff
@@ -0,0 +1,93 @@
+import { Image, View, StyleSheet } from "react-native";
+import { Athlete } from "../../interfaces/athlete";
+import { theme } from "../../theme/theme";
+import { AnimatedFAB, Text } from "react-native-paper";
+import { ImageGrid } from "../ImageGrid/ImageGrid";
+import { useScrollOffset } from "../../hooks/useScrollOffset/useScrollOffset";
+
+const styles = StyleSheet.create({
+  imageContainer: {
+    paddingTop: theme.spacing.md,
+    marginBottom: theme.spacing.md,
+  },
+  imageLabel: {
+    color: theme.colors.gray.DEFAULT,
+    marginBottom: theme.spacing.xs,
+  },
+  imageItem: {
+    height: 300,
+    width: "100%",
+    marginTop: theme.spacing.xs,
+  },
+  imageGrid: {
+    marginTop: theme.spacing.xs,
+  },
+  addButton: {
+    position: "absolute",
+    right: theme.spacing.xxs,
+    marginBottom: theme.spacing.lg,
+  },
+});
+
+type AthleteImagesProps = {
+  athlete: Partial<Athlete>;
+  isEditMode?: boolean;
+  onAddBtnPress?: () => void;
+};
+
+export const AthleteImages = ({
+  athlete,
+  isEditMode = false,
+  onAddBtnPress = undefined,
+}: AthleteImagesProps) => {
+  const { isTopEdge, calcScrollOffset } = useScrollOffset(true);
+
+  const addBtn = isEditMode && (
+    <AnimatedFAB
```

Comment:
This doesn't animate but I am not sure it needs to be animated. I think the animated fabs are best used when floating somewhere on the screen. This could be a regular button I think.